scripts/updateNugetCredentials.ps1

```diff
@@ -1,8 +0,0 @@
-param([Parameter(Mandatory=$true)][string]$username, [Parameter(Mandatory=$true)][string]$apiToken, [Parameter(Mandatory=$true)][string]$nugetFileAbsolutePath)
```

Comment:
Deleting this file breaks these workflows as well and they will need similar updates the file above.

- https://github.com/microsoft/kiota/blob/main/.github/workflows/serialization-dotnet-json.yml
- https://github.com/microsoft/kiota/blob/main/.github/workflows/authentication-dotnet-azure.yml